### PR TITLE
Bump master to v1.39.0

### PR DESCRIPTION
**Pull request by yashykt**

### CODE REVIEW

**Review by srini100 — APPROVED** [review]

File: BUILD [hunk 1]
```diff
@@ -370,6 +370,7 @@ grpc_cc_library(
         "src/cpp/common/tls_credentials_options_util.h",
         "src/cpp/server/secure_server_credentials.h",
     ],
+    defines = ["GRPC_XDS_USER_AGENT_SUFFIX=C++"],
```
Comment (markdroth) [inline BUILD:373]: How did this get in here?  It doesn't seem to have anything to do with the rest of this PR, and I don't think it's actually useful for anything.  We aren't planning to set the xDS user agent suffix for C++ at all, and even if we were, this is the wrong macro name.

Can you please remove this?  Thanks!

Comment (markdroth) [reply BUILD:373]: CC @apolcyn

Comment (yashykt) [reply BUILD:373]: huh! I've no idea how this got in. https://github.com/grpc/grpc/pull/26486

Comment (srini100) [reply BUILD:373]: I thought this was intentional for the user agent work we were planning to get into the previous release.

**Pull request closed without merging**